### ISSUE TITLE
Visual indication of focus for power toggle button is no where near clear enough

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#3471]
### Bug description

Visual indication of focus for power toggle button is no where near clear enough when the button is "on".

### Steps to reproduce

In the realtime effects panel, tab to a power toggle button, and if necessary press it so that it's "on"

### Expected behavior

The visual indication of focus should be clear

### Actual behavior

![image](https://user-images.githubusercontent.com/11887021/187216904-fcd7877b-48e1-4cbf-ba87-45696b6cda34.png)

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by LWinterberg** [comment]
I'd argue this is the case for all of the buttons.
![selection tool](https://user-images.githubusercontent.com/87814144/187230993-d53b7da8-29fb-4ee7-8fcb-2c11ea394055.png)

![input meter](https://user-images.githubusercontent.com/87814144/187231104-7d32ec1a-ddb3-459e-972b-2e3ad3aae797.png)

![snap to](https://user-images.githubusercontent.com/87814144/187231325-eb36c228-1ef3-4f0e-b172-0547b92d1541.png)

I keep losing it if I don't a) already know where it is and b) know where it's going to go...

**Comment by DavidBailes** [comment]
The buttons on the playback/recording toolbars are definitely not as clear as they should be.
For the others, I agree it's not that clear, but it's about what you'd expect for these controls which come from a fairly old set gui controls.

**Comment by LWinterberg** [comment]
Hello! We just [switched development to Audacity 4](https://forum.audacityteam.org/t/master-has-been-switched-to-4-0-0-development/120444). We've redone, or plan on redoing, various things from scratch, including the entire UI, and the part this bug is about. As such, I'm closing this bug as it will be fixed there and we won't be fixing it for the final releases of Audacity 3.
